4_cv_basics/7_blob_detection/README.md

```diff
@@ -0,0 +1,152 @@
+# Blob Detection
+
+BLOB stands for Binary Large Object
+
+Informally a blob is a region of an image in which some properties like intensity or color are approximately constant.
+
+## Table of Contents
+
+- [Understanding the code](#understanding-the-code)
+- [Detecting the blob](#detecting-the-blob)
```

Comment:
Can be more informative if the contents in the Readme can be divided into the different topic sections similar to [this](https://github.com/SRA-VJTI/Pixels_Seminar/blob/main/3_cv_basics/6_blob_detection.ipynb)